flexmeasures/data/services/sensors.py

```diff
@@ -7,23 +7,32 @@
 
 
 def get_sensors(
-    account_name: str | None = None,
+    account: Account | int | str | None = None,
+    sensor_ids: list[int] | None = None,
```

Comment:
Not a good name imho. How about "sensor_id_whitelist" ? "sensor_id_filter"?'

I tend to think we don't need it.
We now have double logic as we already do`  if sensor.id in sensor_ids` in sensors_to_show()

Reply:
I removed the double logic by removing the now redundant if statement in `sensors_to_show()`.

Reply:
I still think the name is not clear. It's used as a whitelist.